tests/jobs/scala/build.sbt

```diff
@@ -1,6 +1,7 @@
 lazy val root = (project in file("."))
   .settings(
     name := "dcos-spark-scala-tests",
+    version := "0.2-SNAPSHOT",
```

Comment:
Don't forget to update the streaming test with the new version.
This is referenced from the Makefile. Do you want to update the version there as well?

Reply:
Yup I updated it on my branch for the streaming test. 
Thanks for pointing out the Makefile, I missed that!